fossology/FOSSologyUI
# fix(header) accesslevel changed to public

## Description

Removed `accesslevel` header from APIs as it is no more present in the back-end.


## How to test

Upload a file to see if error persists (CORS error) 

Continuation of #208 
@GMishx  please do have a look
